app/routes/account-linking/entitlements.js

```diff
@@ -30,14 +30,21 @@ class Entitlements {
         productId,
         publicationId: this.publicationId,
         timestamp: Date.now()
-      }), "utf8").toString("base64")
+      }), "utf8").toString("base64");
+
```

Comment:
Add a comment to say that this should be the same PPID that will be used for Subscription Linking

Reply:
Added in the subsequent commit. Thanks for this suggestion!